sci-libs/ceres_solver/ceres_solver-2.0.0.recipe

```diff
@@ -0,0 +1,112 @@
+SUMMARY="A large scale non-linear optimization library"
+DESCRIPTION="Ceres Solver 1 is an open source C++ library for modeling and \
+solving large, complicated optimization problems. It can be used to solve \
+Non-linear Least Squares problems with bounds constraints and general \
+unconstrained optimization problems. It is a mature, feature rich, and \
+performant library that has been used in production at Google since 2010."
+HOMEPAGE="http://ceres-solver.org/"
+COPYRIGHT="2015 Google Inc."
+LICENSE="BSD (3-clause)
+	Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ceres-solver/ceres-solver/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="2ab0348e0f65fdf43bebcd325a1c73f7e8999691ee75e2a2981281931c42e9fa"
+SOURCE_DIR="ceres-solver-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	ceres_solver$secondaryArchSuffix = $portVersion
+	lib:libceres$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libamd$secondaryArchSuffix
+	lib:libcamd$secondaryArchSuffix
+	lib:libccolamd$secondaryArchSuffix
+	lib:libcholmod$secondaryArchSuffix
+	lib:libcolamd$secondaryArchSuffix
+	lib:libglog$secondaryArchSuffix
+	lib:liblapack$secondaryArchSuffix
+	lib:libmetis$secondaryArchSuffix
+	lib:libopenblas$secondaryArchSuffix
+	lib:libspqr$secondaryArchSuffix
+	lib:libsuitesparseconfig$secondaryArchSuffix
+	lib:libtbb$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	ceres_solver${secondaryArchSuffix}_devel = $portVersion
+	devel:libceres$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	ceres_solver$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:eigen$secondaryArchSuffix
+	devel:libamd$secondaryArchSuffix
+	devel:libcamd$secondaryArchSuffix
+	devel:libccolamd$secondaryArchSuffix
+	devel:libcholmod$secondaryArchSuffix
+	devel:libcolamd$secondaryArchSuffix
+	devel:libglog$secondaryArchSuffix
+	devel:liblapack$secondaryArchSuffix
+	devel:libmetis$secondaryArchSuffix
+	devel:libopenblas$secondaryArchSuffix
+	devel:libspqr$secondaryArchSuffix
+	devel:libsuitesparseconfig$secondaryArchSuffix
+	devel:libtbb$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage ceres_solver$secondaryArchSuffix \
+	$libDir/libceres.so.$portVersion
+
+BUILD()
+{
+	mkdir -p build && cd "$_"
+
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
```

Comment:
isn't CMAKE_INSTALL_PREFIX part of cmakeDirArgs?

Reply:
IIRC they don't use cmakeDirArgs (GNUInstallDirs)

Reply:
> IIRC they don't use cmakeDirArgs (GNUInstallDirs)

This recipe uses cmakeDirArgs